student_explorer/cron.py

```diff
@@ -2,6 +2,8 @@
 
 from __future__ import print_function #python 3 support
 
+from __future__ import absolute_import #py3 management conflict in cron
```

Comment:
This was the line needed to get this to work in management. 

management conflicts with management in django-cron
https://github.com/Tivix/django-cron/tree/master/django_cron

https://stackoverflow.com/questions/1224741/trying-to-import-module-with-the-same-name-as-a-built-in-module-causes-an-import